lib/constants/decorations.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+InputDecoration inputDecor({String hint, IconData prefix, bool obscure, IconData suffix, Function toogle}) {
```

Comment:
This does not need to be refactored

Reply:
I changed some functionality of it too as it was using two bools before and doing an and operation on them now it only takes one and removed setState from it.